README.md

```diff
@@ -28,7 +28,7 @@ advanced information on [development-instructions.md](development-instructions.m
 Run these commands on your terminal:
 
 ```bash
-git clone git@github.com:fititnt/uwazi-docker.git
+git clone https://github.com/huridocs/uwazi.git
```

Comment:
As you commented and already tested here https://github.com/fititnt/uwazi-docker/issues/15#issue-369692883, this line would be 

```
git clone https://github.com/fititnt/uwazi-docker.git
```

Reply:
Stupid me. Sorry, got distracted and copied the wrong line. Doing too many things in parallel! Thanks for the headsup!